src/Chapter2.hs

```diff
@@ -391,6 +428,22 @@ isZero 0 = True
 isZero n = False
 @
 
+{-
+TODO: did not know underscore is called hole. Do we also use this term in other languages?
```

Comment:
I'm afraid I can't certainly say that. `_` is called hole as you can put it as the placeholder to fill later. It is also called wildcard sometimes.